build.sh

```diff
@@ -0,0 +1,5 @@
+podman build -f build/dockerfiles/linux-musl.Dockerfile -t linux-musl-amd64 .
+podman build -f build/dockerfiles/linux-libc-ubi8.Dockerfile -t linux-libc-ubi8-amd64 .
+podman build -f build/dockerfiles/linux-libc-ubi9.Dockerfile -t linux-libc-ubi9-amd64 .
+podman build -f build/dockerfiles/assembly.Dockerfile -t harbor.weebo.fr/batleforc/che-code:latest .
+podman push harbor.weebo.fr/batleforc/che-code:latest
```

Comment:
harbor.weebo.fr ?
Do we need this file in general or it was added accidentally

Reply:
It was a pure mistake, half of the commit hasn't been pushed and I didn't see it :/